forge-gui/res/cardsfolder/upcoming/nanogene_conversion.txt

```diff
@@ -0,0 +1,7 @@
+Name:Nanogene Conversion
+ManaCost:3 U
+Types:Sorcery
+A:SP$ Pump | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Choose target creature you control | SubAbility$ DBRepeatEach | StackDescription$ Each other creature becomes a copy of {c:Targeted} until end of turn, except those creatures aren't legendary | SpellDescription$ Choose target creature you control. Each other creature becomes a copy of that creature until end of turn, except it isn't legendary.
+SVar:DBRepeatEach:DB$ RepeatEach | RepeatCards$ Creature.NotDefinedTargeted | RepeatSubAbility$ DBClone
+SVar:DBClone:DB$ Clone | Defined$ Targeted | CloneTarget$ Remembered | NonLegendary$ True | Duration$ UntilEndOfTurn
```

Comment:
Probably should add loop functionality to CloneEffect so we don't need repeats for cards like this

Reply:
gonna merge this and maybe add the new functionality on new MR